content/news/025/index.md

```diff
@@ -67,6 +67,26 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [vange-rs]
+
+![Vange-rs on wgpu-0.10](./vangers-wgpu0.10.png)
```

Comment:
could you please add a one-sentence description of the project as a reminder for readers?

Reply:
Thanks for the advice!